v3/src/v2/codap-v2-types.ts

```diff
@@ -554,23 +723,39 @@ export interface ICodapV2TextComponent extends ICodapV2BaseComponent {
 export const isV2TextComponent = (component: ICodapV2BaseComponent): component is ICodapV2TextComponent =>
               component.type === "DG.TextView"
 
-export type CodapV2Component = ICodapV2CalculatorComponent | ICodapGameViewComponent | ICodapV2GraphComponent |
-                                ICodapV2GuideComponent | ICodapV2MapComponent | ICodapV2SliderComponent |
-                                ICodapV2TableComponent | ICodapV2TextComponent | ICodapV2WebViewComponent
+export type CodapV2Component =
+  ICodapV2CalculatorComponent |
+  ICodapV2CaseCardComponent|
+  ICodapGameViewComponent |
```

Comment:
```suggestion
  ICodapV2GameViewComponent |
```
for consistency with the other names